user-service/src/main/java/com/recipe/app/user/service/UserServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.recipe.app.user.service;
+
+import com.recipe.app.user.model.User;
+import com.recipe.app.user.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    private final UserRepository userRepository;
+
+    @Autowired
+    UserServiceImpl(UserRepository userRepository) {
```

Comment:
don't know if I didn't have the public on my constructor but it should have been there